Surface_mesh_deformation/include/CGAL/Surface_mesh_deformation.h

```diff
@@ -43,14 +44,17 @@
 
 namespace CGAL {
 
-/// \ingroup PkgSurfaceMeshDeformationRef
-///@brief Deformation algorithm type
-enum Deformation_algorithm_tag
-{
-  ORIGINAL_ARAP,   /**< use original as-rigid-as possible algorithm */
-  SPOKES_AND_RIMS, /**< use spokes and rims version of as-rigid-as possible algorithm */
-  SRE_ARAP         /**< use smooth rotation enhanced As-rigid-as-possible */
-};
+#ifdef DOXYGEN_RUNNING
+  /// \ingroup PkgWeightsRefARAPWeights
```

Comment:
Looks like the same problem as seen in Weights/include/CGAL/Weights/arap_weights.h (see: https://github.com/CGAL/cgal/pull/8329/files/47b7f67983cd56aa1b82efdea0c8cecdc39411e0..688f4a1e2b17507d06ec777d0c2d0ed4275a3512#r1847995747)